examples/Button.stories.svelte

```diff
@@ -20,6 +21,7 @@
     tags: ['autodocs'],
     args: {
       onclick: onclickFn,
+      children: 'Click me' as unknown as Snippet<[]>
```

Comment:
Note sure if this is cleanst way but could find no examples of passing string as a children prop

Reply:
I've changed to `any` instead